### ISSUE TITLE
iframe accessibility

### ISSUE THREAD

**Issue by TomPridham** [braintree/braintree-web#434]
### General information

* SDK version: 3.45.0 
* Environment: both 
* Browser and OS: at least Chrome and MacOs

### Issue description
related: https://github.com/braintree/braintree-web/issues/374
there are two iframes being loaded by the braintree/paypal client. one is related to your FraudNet data-collector with `title="ppfniframe"`. i wasn't clear where it was coming from after that. the other, is being injected by your frameService that does not have a title.

it seems like all they contain is javascript, with no content that would need to be accessed by a user. the one without a title should be given a title. if they both only contain javascript or other non-interactive/non-readable content they should be given the `aria-hidden` attribute so that they are unnavigable for people using screen readers.

i'll open a pr with a fix for the second one, but would appreciate if you could fix the other one wherever that lives or point me to it so i can open a pr there as well.

https://dequeuniversity.com/tips/provide-iframe-titles

**Comment by crookedneighbor** [comment]
The fraudnet iframe is created by a PayPal library. We'll escalate it to that team. Feel free to open a PR for the other one.

**Comment by crookedneighbor** [comment]
Your change is out in 3.46.0. The Fraudnet team has been alerted about the issue on their end, and they are working on it.

Thanks again!
